### PR TITLE
[C1-777] fix jobId retrieval in timestampHelperFactory

### PR DESCRIPTION
**Pull request by VLDCNDN**
#### Description

Resolved an issue where `jobId` was not being correctly extracted from context parameter, leading to jobs only triggering once and not comply to the `repeat_interval`. The PR updates argument handling to get `jobId` via args parameter, this will ensure that jobs are rescheduled as expected.

----
#### Changes
_List the main changes in this PR._

----

#### Testing Instructions
_Outline the steps to test or reproduce the PR._

----

#### Screenshots
_(if appropriate, they help the reviewer visualise the changes)_

----

### CODE REVIEW

**Review by ish-bindra — COMMENTED** [review]
We should fix the types if we can. I can pair and help out if needed.

**Review by ish-bindra — APPROVED** [review]

**Review by tjcoyoca — DISMISSED** [review]

**Review by tjcoyoca — APPROVED** [review]

File: packages/scheduler-prisma/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@steveojs/scheduler-prisma",
-  "version": "6.0.1",
+  "version": "6.0.2-beta",
```
Comment (johndagostino) [inline packages/scheduler-prisma/package.json:3]: did you publish it to test locally?

we should move away from doing that and just using `yarn link`

File: packages/scheduler-sequelize/src/helpers.ts [hunk 2]
```diff
@@ -144,17 +147,12 @@ const updateFinishTask = async (job?: JobInstance | null) => {
...
-    if (!context) {
-      try {
-        return task(args, context);
-      } catch (e) {
-        return null;
-      }
-    }
-
-    const jobId = context.job?.id;
+    const jobId = args.context?.job?.id ?? context?.job?.id;
```
Comment (johndagostino) [inline packages/scheduler-sequelize/src/helpers.ts:155]: does the full job get serialized into the meta of the task being run?

File: packages/scheduler-prisma/package.json (hunk 1 above)
Comment (VLDCNDN) [reply packages/scheduler-prisma/package.json:3]: Noted

File: packages/scheduler-sequelize/src/helpers.ts (hunk 2 above)
Comment (VLDCNDN) [reply packages/scheduler-sequelize/src/helpers.ts:155]: yes, I attached the image for ref
image (1)

Comment (ish-bindra) [reply packages/scheduler-sequelize/src/helpers.ts:155]: We should make the job object leaner. 
https://github.com/ordermentum/steveo/blob/C1-777-fix-jobid/packages/scheduler-prisma/src/index.ts#L334-L336

```typescript
await task(item.data, {
       job: {
          id: item.id
       }
});
```
The above change will make it leaner and should not break things.
We'll need to do a sweep of what service uses the context object (if any). 

Not saying we do it here, we should def unblock this. A new ticket also works.

**Review by ish-bindra — APPROVED** [review]
As per my comment, we should do a follow up to see if we can make the context leaner.

File: packages/scheduler-sequelize/src/helpers.ts (hunk 2 above)
Comment (VLDCNDN) [reply packages/scheduler-sequelize/src/helpers.ts:155]: done creating a ticket

**Review by ish-bindra — APPROVED** [review]

**Review by johndagostino — APPROVED** [review]

File: packages/scheduler-prisma/package.json (hunk 1 above)
Comment (sugendran) [reply packages/scheduler-prisma/package.json:3]:
```suggestion
  "version": "6.0.2",
```

@VLDCNDN this needs to drop the `-beta`

**Pull request merged**